### DISCUSSION TITLE
Output Folder

### DISCUSSION THREAD

**Discussion by justinliuyuwang** [McIntosh-Lab/tvb-ukbb#182] [Ideas]
"Thanks a lot for this great tool!
I was wondering whethere there is a simple script or edit I have missed so that all the outputs from a patient, e.g. sub-01, are not outputted inside the sub-01 folder but rather in a completely separate directory chosen by the user:
e.g.
-derivatives
-- sub-01_preprocessed
--- ses-*
---- all output folders from tvb pipeline"

- @marcellamontagnese

**Comment by justinliuyuwang** [comment]
Hi Marcella! 

Thanks for your post!
Unfortunately, there is no script or setting you can toggle to get the outputs to a completely separate directory.
Maybe try this once you're done processing your subjects.:


```
#hardcode session # below
session=2

for dir in sub-*; do
    if [ -d "$dir" ]; then
        mv "$dir"/!(rawdata) "derivatives/${dir}_preprocessed/ses-${session}/"
    fi
done
```

**Reply by marcellamontagnese** [reply]
Thanks for your quick reply! @justinliuyuwang
